pkg/apis/internal.admin.acorn.io/v1/resources.go

```diff
@@ -9,6 +9,13 @@ import (
 
 var ErrExceededResources = fmt.Errorf("quota would be exceeded for resources")
 
+const Unlimited = -1
```

Comment:
Does this need to be exported? Is it used elsewhere? I'm assuming the answer is "yes", but figured I'd ask anyway.

Reply:
Yep, it will be used elsewhere, namely in Manager.